### PR TITLE
Refine GetResRule API and GetRules API in flow/circuitbreaker/hotspot/system module

### PR DESCRIPTION
**Pull request by louyuting**
### Describe what this PR does / why we need it
Refine GetResRule func and GetRules func in flow/circuitbreaker/hotspot/system module

### Special notes for reviews

### CODE REVIEW

File: core/circuitbreaker/rule_manager.go [hunk 1]
```diff
@@ -69,12 +69,35 @@ func init() {
 	}
 }
 
-func GetResRules(resource string) []*Rule {
+// GetResRules returns specific resource's rules based on copy.
+// It doesn't take effect for circuit breaker module if user changes the rule.
+// GetResRules need to compete circuit breaker module's global lock and the high performance losses of copy,
+// 		reduce or do not call GetResRules frequently if possible
+func GetResRules(resource string) []Rule {
```
Comment (sczyh30) [inline core/circuitbreaker/rule_manager.go:76]: How about `GetRulesOfResource`?

Comment (louyuting) [reply core/circuitbreaker/rule_manager.go:76]: It's ok for me.

**Review by sczyh30 — APPROVED** [review]
LGTM

**Comment by sczyh30** [comment]
Thanks!

**Pull request merged**